.github/workflows/linux-tests.yml

```
@@ -21,31 +21,44 @@ jobs:
       fail-fast: false
       matrix:
         PYTHON_VERSION: ['3.7', '3.8', '3.9']
+        INSTALL_TYPE: ['conda', 'pip']
     steps:
       - name: Checkout branch
         uses: actions/checkout@v1
-      - name: Install System Packages
+      - name: Install System Packages (conda)
+        if: matrix.INSTALL_TYPE == 'conda'
         run: |
           sudo apt-get update
           sudo apt-get install libegl1-mesa
+      - name: Install System Packages (pip)
+        if: matrix.INSTALL_TYPE == 'pip'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -qq pyqt5-dev-tools
       - name: Install Conda
         uses: conda-incubator/setup-miniconda@v2
         with:
            activate-environment: test
            auto-update-conda: false
            auto-activate-base: false
            python-version: ${{ matrix.PYTHON_VERSION }}
-      - name: Install dependencies
+      - name: Install dependencies with conda
+        if: matrix.INSTALL_TYPE == 'conda'
         shell: bash -l {0}
         run: conda env update --file requirements/environment.yml
+      - name: Install dependencies with pip
+        if: matrix.INSTALL_TYPE == 'pip'
+        shell: bash -l {0}
+        run: pip install -r requirements/requirements.txt
```

```suggestion
        run: |
          pip install -e .[test]
          pip install PyQt5 coveralls pytest-cov
```

This will avoid declaring an extra `requirements.txt` file and will also test that our `setup.py` is working as expected.